data/data/bootstrap/baremetal/files/etc/containers/systemd/image-customization.container

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Customized Machine OS Image Server
+BindsTo=ironic-volume.service
+Requires=build-ironic-env.service extract-machine-os.service
+PartOf=ironic.service
+Wants=network-online.target
+After=network-online.target ironic-volume.service build-ironic-env.service extract-machine-os.service
+
+[Container]
+ContainerName=image-customization
+Image=$CUSTOMIZATION_IMAGE
+Exec=/image-customization-server --nmstate-dir=${NMSTATE_DIR} --images-publish-addr=http://0.0.0.0:8084
+Network=host
+PodmanArgs=--privileged --secret pull-secret,mode=400
+Volume=${NMSTATE_DIR}/:${NMSTATE_DIR}/
+Volume=ironic.volume:/shared:z
+Volume=/etc/containers:/tmp/containers:z
+Environment="DEPLOY_ISO=/shared/html/images/ironic-python-agent.iso"
+Environment="DEPLOY_INITRD=/shared/html/images/ironic-python-agent.initramfs"
+Environment="IRONIC_BASE_URL=${IRONIC_BASE_URL}"
```

Comment:
This is not coming from ironic.env, is it? I find tracing the variables quite confusing...